packages/darklang/languageTools/semanticTokens.dark

```diff
@@ -340,6 +340,23 @@ module Darklang =
               [ makeToken symbolCloseParen TokenType.Symbol ] ]
             |> Stdlib.List.flatten
 
+          // Person { name = "Alice" }
+          | ERecord(range, name, fields, symbolOpenBrace, symbolCloseBrace) ->
+            let fields =
+              fields
+              |> Stdlib.List.map (fun (_, fieldName, value) -> Expr.tokenize value)
+              |> Stdlib.List.flatten
```

Comment:
although we're not testing this quite yet, we might as well make this handle `fieldName: value` rather than just `value`